### PR TITLE
feat: update Chrome browser binaries

### PR DESCRIPTION
**Pull request by OrKoN**
Drive-by: separated tests to be by browser. describe/it names should be improved in a follow-up.

### CODE REVIEW

File: packages/browsers/src/httpUtil.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function httpRequest(
     protocol: url.protocol,
     hostname: url.hostname,
     port: url.port,
-    path: url.pathname,
+    path: url.pathname + url.search,
```
Comment (Lightning00Blade) [inline packages/browsers/src/httpUtil.ts:51]: Nit: I think this should be `url.href` as that is the full url
```suggestion
    path: url.href,
```

File: packages/browsers/test/src/chrome/cli.spec.ts [hunk 2]
```diff
@@ -92,40 +74,14 @@ describe('CLI', function () {
...
-      '@puppeteer/browsers',
-      'install',
-      `firefox@latest`,
-      `--path=${tmpDir}`,
-      '--platform=linux',
-    ]);
-  });
-
-  it('should download latest Chrome binaries', async () => {
+  it.skip('should download latest Chrome binaries', async () => {
```
Comment (Lightning00Blade) [inline packages/browsers/test/src/chrome/cli.spec.ts:85]: Nit: Add comment why skipped.

**Review by Lightning00Blade — APPROVED** [review]

**Comment by swissspidy** [comment]
I just updated to Puppeteer 19.8.1 and I think this change here caused a regression.

With `PUPPETEER_PRODUCT=chrome npm install puppeteer`, it downloads Chromium 1108766 to `~/.cache/puppeteer/chromium`, but then when running Puppeteer, the browser fetcher is looking for `~/.cache/puppeteer/chrome/mac-1108766` which of course doesn't exist. Then I get `Could not find Chromium (rev. 1108766)` errors.

Is that a known issue and perhaps #9937 solves this?

When I downgrade to Puppeteer 19.8.0, Chromium is indeed downloaded to `~/.cache/puppeteer/chrome` and all works.

**Comment by OrKoN** [comment]
@swissspidy thanks for the report! It looks like this is an unintended change. I am looking into it.

**Comment by OrKoN** [comment]
One of the important changes for compatibility was marked as a chore so the browsers package version didn't get bumped: https://github.com/puppeteer/puppeteer/pull/9939 should address it.

**Pull request merged**